### PR TITLE
Classname mixin & Title with variable tag

### PR DESCRIPTION
**Pull request by Jones-S**
– Remove Dashboard
– Add classname Mixin to write modifiers (tbd: is there a nicer way than to register all modifiers? > Maybe we could just register them within props, if they are not present? Possible?)
– Add example for title with variable h-level and with modifier

### CODE REVIEW

File: template/client/components/Title/style.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+$c: 'Title';
+$c-HighlightColor: rgb(237, 48, 30);
+
+.#{$c} {
+  &--h1 {
+    @include fontStyleXLarge;
+  }
+
+  &--h2 {
```
Comment (marcoeh) [inline template/client/components/Title/style.scss]: `&--h2, &--h3, &--h4, &--h5, &--h6 {` ?

File: template/client/components/Title/style.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+@import '../../styles/global';
+
+$c: 'Title';
+$c-HighlightColor: rgb(237, 48, 30);
```
Comment (marcoeh) [inline template/client/components/Title/style.scss]: Do we have global color vars defined somewhere?

File: template/client/views/Home/index.js [hunk 3]
```diff
@@ -18,9 +20,11 @@ export default {
 
     return (
       <div class='page'>
+        <Title level='h1'> H1 Title</Title>
```
Comment (marcoeh) [inline template/client/views/Home/index.js]: I'd rather say `<Title level='h1'>Page Title</Title>`

**Comment by Jones-S** [comment]
Ah btw: we need a proper removal of the dashboard. So far I just uncommented it. @d-simon maybe you can remove it rather quickly, regarding you having installed it in the first place.

File: template/client/views/Home/index.js [hunk 4]
```diff
@@ -18,9 +20,11 @@ export default {
...
         <p>
           To get started, edit files in <code>./client</code> and save.
         </p>
+        <Title level='h2' highlighted>Modified H2 Title</Title>
```
Comment (marcoeh) [inline template/client/views/Home/index.js]: Would it make sense to provide an array like `<Title level='h2' modifiers=['highlighted']>Modified H2 Title</Title>`? Dunno if it's proper syntax…

**Review by marcoeh — COMMENTED** [review]
I like where this is going 👍

**Comment by marcoeh** [comment]
Dashboard: Wrong PR 🙃

File: template/client/components/Title/style.scss (hunk 2 above)
Comment (Jones-S) [reply template/client/components/Title/style.scss]: Yes. I was lazy

**Pull request merged**